astroid/rebuilder.py

```diff
@@ -1607,7 +1607,7 @@ def visit_attribute(
             if not isinstance(parent, nodes.ExceptHandler):
                 self._delayed_assattr.append(newnode)
         else:
-            if sys.version_info >= (3, 8):
+            if sys.version_info >= (3, 8):  # pylint: disable=else-if-used
```

Comment:
As discussed in the other issue, I don't think it's a false positive.

Reply:
Even if not a false-positive, I would recommend to keep the `pylint: disable` as this preserves the symmetry with all other cases and will make removing the check once `3.8` is the default much easier.